src/model/outputChannel.ts

```
@@ -68,7 +68,8 @@ export default class BufferChannel implements OutputChannel {
     let { nvim } = this
     let winid = await nvim.call('win_getid')
     nvim.pauseNotification()
-    nvim.command(`tab drop output:///${this.name}`, true)
+    const escapedName = await nvim.call('fnameescape', this.name)
+    nvim.command(`tab drop output:///${escapedName}`, true)
```

Need escape for `bufname` property and use bufrname here,  we could also throw error for invalid characters for channel name at https://github.com/neoclide/coc.nvim/blob/master/src/channels.ts#L39, I think that limit the characters to `[\w\s-]` is good enough.